src/app/clusters/general-diagnostics-server/general-diagnostics-server.cpp

```diff
@@ -195,11 +195,24 @@ class GeneralDiagnosticsDelegate : public DeviceLayer::ConnectivityManagerDelega
     }
 
     // Gets called when the device has been rebooted.
-    void OnDeviceRebooted() override
+    void OnDeviceRebooted(uint8_t bootReason) override
     {
         ChipLogProgress(Zcl, "GeneralDiagnosticsDelegate: OnDeviceRebooted");
 
         ReportAttributeOnAllEndpoints(GeneralDiagnostics::Attributes::BootReasons::Id);
+
+        for (auto endpoint : EnabledEndpointsWithServerCluster(GeneralDiagnostics::Id))
```

Comment:
Spec requires GeneralDiagnostics to exist and only exist on endpoint 0, fwiw....

Reply:
Ok, fixed